modules/jwplayerVideoProvider.js

```diff
@@ -92,6 +94,20 @@ export function JWPlayerProvider(config, jwplayer_, adState_, timeState_, callba
     const adConfig = config.advertising || {};
     supportedMediaTypes = supportedMediaTypes || utils.getSupportedMediaTypes(MEDIA_TYPES);
 
+    if (height === null) {
+      height = utils.getPlayerHeight(player, config);
+    }
+
+    if (width === null) {
+      width = utils.getPlayerWidth(player, config);
+    }
+
+    if (config.aspectratio && !height && !width) {
+      const size = utils.getPlayerSizeFromAspectRatio(config, divId);
```

Comment:
is this right? the signature reads `getPlayerSizeFromAspectRatio(player, config)`.

Reply:
forgot to commit, thanks!